src/wrappers/themis/rust/src/secure_cell.rs

```diff
@@ -25,36 +91,85 @@ use bindings::{
     themis_secure_cell_decrypt_token_protect, themis_secure_cell_encrypt_context_imprint,
     themis_secure_cell_encrypt_seal, themis_secure_cell_encrypt_token_protect,
 };
-use error::{Error, ErrorKind, Result};
-use keys::KeyBytes;
-use utils::into_raw_parts;
+
+use crate::error::{Error, ErrorKind, Result};
+use crate::keys::KeyBytes;
+use crate::utils::into_raw_parts;
 
 /// Basic Secure Cell.
 ///
-/// This is modeless, basic cell. Specific operation mode must be selected by one of the calls:
-///
-/// * `seal()`
-/// * `token_protect()`
-/// * `context_imprint()`
+/// This is modeless, basic cell. First you create a `SecureCell` object with a key and an optional
+/// context then you select the desired operation mode and your Secure Cell is ready to go.
 pub struct SecureCell {
```

Comment:
as @Lagovas mentioned, from security point of view, `SecureCell` shouldn't take `context` as init param. Context is an additional parameter in encryption function which increase security guarantees for messages encrypted with same key.

Reply:
Yeah... I thought about that when I started working on integration tests and studied other wrappers and their API more carefully. I doesn't make much sense to keep the user context the same for all operations – then it can be a part of the key. However, changing the API looked like a hassle so I never got to that (just [left some TODOs](https://github.com/ilammy/rust-themis/blob/842aaacebb454aeb7cc8f8162387ce907df7c52d/tools/scell_token_string_echo.rs#L46) in my WIP branches).

I will unify the API to be the same as the other language wrappers, but I would like to do this in a separate pull request, after this one is merged and Rust wrapper is integrated into the build system. That way we'd be able to see that the changes do not break anything, the code still compiles and the tests pass.

Please rest assured, this change *is* on my (private) task list.

Reply:
ok, makes sense